node/docs/stepbystep.md

```diff
@@ -294,10 +303,16 @@ tmr.setInput('samplebool', 'true');
 // provide answers for task mock
 let a: ma.TaskLibAnswers = <ma.TaskLibAnswers>{
     "which": {
-        "echo": "/mocked/tools/echo"
+        "echo": "/mocked/tools/echo",
+        "cmd": "/mocked/tools/cmd"
     },
     "exec": {
-        "/mocked/tools/echo Hello, from task!": {
+        "/mocked/tools/echo \"Hello, from task!\"": {
```

Comment:
working through unit tests still... i might need to remove the double quotes from within the string